ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -552,20 +662,21 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 7AFA3C8E1D35360C0083082E /* Release.xcconfig */;
 			buildSettings = {
+				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = Runner/Runner.entitlements;
-				CODE_SIGN_IDENTITY = "iPhone Distribution";
-				CODE_SIGN_STYLE = Manual;
+				CODE_SIGN_IDENTITY = "Apple Development";
+				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = "$(FLUTTER_BUILD_NUMBER)";
-				DEVELOPMENT_TEAM = MYEC2JCSBB;
+				DEVELOPMENT_TEAM = "";
```

Comment:
👀

Reply:
Don't care here - this seems to be the "Profile" signing profile. I am curious if CI still works, as a new Provisioning Profile was needed.

Reply:
If it's an irrelevant change, let's not commit it to avoid noise from changing it around all the time. 

Same of all changes to this file here.